client/packages/common/src/intl/locales/tet/replenishment.json

```diff
@@ -37,7 +37,7 @@
   "label.draft": "Draf",
   "label.finalised": "Finaliza Ona",
   "label.max": "Maximu",
-  "label.max-months-of-stock": "MOS Maximu",
```

Comment:
These are again best guesses at translations. (not reviewed by a native speaker)